app/javascript/dashboard/components/widgets/conversation/bubble/Actions.vue

```diff
@@ -252,6 +254,14 @@ export default {
       position: absolute;
       right: var(--space-small);
       white-space: nowrap;
+      &.delivered {
+        right: var(--space-medium);
+      }
+    }
+    .read-tick {
+      position: absolute;
+      bottom: var(--space-mini);
```

Comment:
Instead of adding a new variable. Let us add a `line-height: 2` inside `.delivered` class
```suggestion
      bottom: var(--space-small);
```

cc: @nithindavid